search/search-client-opensearch/src/test/java/io/camunda/search/os/transformers/index/IndexAliasResponseTransformerTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.os.transformers.index;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import io.camunda.search.clients.index.IndexAliasResponse;
+import io.camunda.search.os.transformers.OpensearchTransformers;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import org.opensearch.client.opensearch.indices.AliasDefinition;
+import org.opensearch.client.opensearch.indices.GetAliasResponse;
+import org.opensearch.client.opensearch.indices.get_alias.IndexAliases;
+
+public class IndexAliasResponseTransformerTest {
+
+  private final OpensearchTransformers transformers = new OpensearchTransformers();
+
+  @Test
+  public void shouldDecodeCorrectly() {
+    final var response =
+        GetAliasResponse.of(
+            b ->
+                b.result(
+                    Map.of(
+                        "index-1",
+                        IndexAliases.of(
+                            bia ->
+                                bia.aliases(
+                                    Map.of(
+                                        "alias-1",
+                                        AliasDefinition.of(
+                                            bad ->
+                                                bad.routing("routing")
+                                                    .indexRouting("index-routing")
+                                                    .searchRouting("search-routing"))))))));
```

Comment:
:fearful: I mean it's test, so I guess whatever, but it's still funky looking :smile:

Reply:
Actually even then, it's a little hard to read. Can we extract just, say, `AliasDefinition` ? I think that will already make it more readable.